clockchain.py

```diff
@@ -27,8 +27,8 @@
 
 # TODO: Make more secure way of retrieving private key
 # TODO: For now this file is in .gitignore
-with open('priv.json') as privkey_file:
-    privkey = json.load(privkey_file)
+#with open('priv.json') as privkey_file:
+#    privkey = json.load(privkey_file)
```

Comment:
how come you commented out the privkey as file thing?

Reply:
As this private key file is not included in the repository, I think we should either remove this code from the main branch (develop on a feature branch) or include a sample file, so that the code on master can be run as soon as it is pulled. As it currently stands, the code won't run because it doesn't find this file, even if you have it set to generate the keys at runtime.